tests/ui/test_non_admin_user.py

```diff
@@ -52,22 +51,18 @@ def finalizer():
     @skipif_ibm_cloud_managed
     @bugzilla("2031705")
     @polarion_id("OCS-4620")
-    def test_create_storageclass_rbd(self, user_factory, login_factory):
-        """Create user"""
-        user = user_factory()
-        sleep(30)
-
-        """ Create RoleBinding """
-        ocp_obj = ocp.OCP()
-        ocp_obj.exec_oc_cmd(
-            f"-n openshift-storage create rolebinding {user[0]} --role=mcg-operator.v4.11.0-noobaa-odf-ui-548459769c "
-            f"--user={user[0]} "
-        )
+    def test_project_admin_obcs_access(self, user_factory, login_factory):
+        """
+        Test if project admin can view the list of OBCs
 
-        """Login using created user"""
+        """
+        user = user_factory()
+        sleep(5)
```

Comment:
It would be good to use explict wait types in UI automation..

Reply:
Thank you, fixed